geant-incubator/Issues/surf-sram/description.xml

```diff
@@ -0,0 +1,186 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo SURF SRAM</en>
+        <nl>Demo SURF SRAM</nl>
```

Comment:
This is not a descriptive name, and the `SRAM` acronym is nowhere explained. Would it not be better to choose a name that is more understandable to users?

Reply:
SRAM stands for SURF Research Access Management, so SURF SRAM is incorrect